src/BladeSDK.ts

```diff
@@ -626,7 +627,12 @@ export class BladeSDK {
      */
     async getKeysFromMnemonic(mnemonicRaw: string, lookupNames: boolean, completionKey?: string): Promise<PrivateKeyData> {
         try {
-            const mnemonic = await Mnemonic.fromString(mnemonicRaw);
+            const mnemonic = await Mnemonic.fromString(mnemonicRaw
+                .toLowerCase()
+                .split(" ")
+                .filter(word => word)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
trim not help if there will be 2 spaces in row between words. this solution helps